### PR TITLE
Feature/gap

### PR DESCRIPTION
**Pull request by rborn**

### CODE REVIEW

**Comment by rborn** [comment]
for #205 , not sure why the previous PR is not there 🧐

File: src/ClassParser.ts [hunk 1]
```diff
@@ -302,6 +302,15 @@ export default class ClassParser {
       if (style) return style;
     }
 
+    if (this.consumePeeked(`gap`)) {
+      style = gap(
+        this.rest,
+        this.context,
+        theme === null || theme === void 0 ? void 0 : theme.gap,
+      );
```
Comment (jaredh159) [inline src/ClassParser.ts:310]: we can simplify this to just: 

```suggestion
      style = gap(this.rest, this.context, theme?.gap);
```

**Review by jaredh159 — CHANGES_REQUESTED** [review]
this looks great, just the one proposed change to tighten up the call site.

thanks for the contribution, again. 👍

File: src/ClassParser.ts (hunk 1 above)
Comment (jaredh159) [reply src/ClassParser.ts:310]: @rborn oh, and can you rebase? it looks like you've got some old commits in there. if you add  an upstream remote, you should be able to run `git pull --rebase upstream master` and eliminate those.

**Comment by rborn** [comment]
@jaredh159 ok, fixed `theme?.gap` and maybe some merge conflicts?  The diff looks clean, maybe we can merge ?

**Comment by jaredh159** [comment]
@rborn code diff looks good, but i'm a bit of a stickler about keeping my commit history tidy, and you've got duplicated commits, and commits from previous PR's in your history, so i'd rather not merge all of that in. do you know how to clean that up with a rebase? if not, can you maybe add me as a collaborator on your fork, and i'll rebase it and then i can merge?

**Comment by rborn** [comment]
@jaredh159 yeah, I managed somehow to do a total mess out of this 😅 Now it should be clean, sorry for this

**Pull request merged**